### PR TITLE
create/edit contact done

### PR DESCRIPTION
**Pull request by sahemant12**
for creating more than 1 contact we have to globally declared no. of contacts

### CODE REVIEW

**Comment by sahemant12** [comment]
suman code dekh dena

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# ds-projects
```
Comment (sumansahoo1) [inline README.md]: code toh shi h, ye kya flatu ka cheez kr rha h

Comment (sahemant12) [reply README.md]: 7 days phele ka hai pta nhi kese ho gya

Comment (sumansahoo1) [reply README.md]: hata de,  jarurat nhi h

**Comment by sumansahoo1** [comment]
jab code kr rha h toh update krne k baad code kiya kr, bole toh the 

kyu nhi kiya h phir ? conflicts aa rha h

**Review by ghost — APPROVED** [review]

**Pull request merged**